.gitignore

```diff
@@ -5,3 +5,4 @@ typings/
 *.tsbuildinfo
 dist/
 .code
+/coverage
```

Comment:
I think this should be `coverage/`. The leading `/` means we're matching on directories or files, not just directories.

Reply:
According to [gitignore documentation](https://git-scm.com/docs/gitignore), the leading `/` means that "the pattern is relative to the directory level of the particular .gitignore file itself." In our case, the leading `/` is not needed, since we want to "match at any level below the .gitignore level."
The last `/` means that we're matching only on directories, which was the missing part indeed.

Adjusted ignoring the `coverage` directory properly.

Reply:
Thanks!